src/test/resources/de.unijena.cheminf.mortar.model.fragmentation.algorithm.ASF/ASF_Butene_Test.mol

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
are all of these files still in use? If not, please remove unused resource files.